cedar-policy-core/src/ast/entity.rs

```diff
@@ -122,11 +155,17 @@ impl EntityUID {
         (self.ty, self.eid)
     }
 
+    /// Get the source location for this `EntityUID`.
+    pub fn loc(&self) -> &Option<Loc> {
```

Comment:
Slightly prefer `Option<&Loc>` in interfaces instead of `&Option<Loc>`, it is easier to work with in some subtle ways.  You can get it with `self.loc.as_ref()` instead of `&self.loc`